get-started/part2.md

```diff
@@ -218,53 +229,105 @@ docker stop 1fa4ab2cf395
 
 ## Share your image
 
-To demonstrate the portability of what we just created, let's upload our build
-and run it somewhere else. After all, you'll need to learn how to push to
+To demonstrate the portability of what we just created, let's upload our built
+image and run it somewhere else. After all, you'll need to learn how to push to
 registries to make deployment of containers actually happen.
 
 A registry is a collection of repositories, and a repository is a collection of
-images -- sort of like a GitHub repository, except the code is already built. An
-account on a registry can create many repositories. The `docker` CLI is
-preconfigured to use Docker's public registry by default.
+images &#8212; sort of like a GitHub repository, except the code is already
+built. An account on a registry can create many repositories. The `docker` CLI
+is preconfigured to use Docker's public registry by default.
 
 > **Note**: We'll be using Docker's public registry here just because it's free
 and pre-configured, but there are many public ones to choose from, and you can
 even set up your own private registry using [Docker Trusted
 Registry](/datacenter/dtr/2.2/guides/).
 
+### Log in with your Docker ID
+
 If you don't have a Docker account, sign up for one at
 [cloud.docker.com](https://cloud.docker.com/). Make note of your username.
 
-Log in your local machine.
+Log in to the Docker public registry on your local machine.
 
 ```shell
 docker login
 ```
 
-Now, publish your image. The notation for associating a local image with a
-repository on a registry, is `username/repository:tag`. The `:tag` is optional,
-but recommended; it's the mechanism that registries use to give Docker images a
-version. So, putting all that together, enter your username, and repo
-and tag names, so your existing image will upload to your desired destination:
+### Tag the image
+
+The notation for associating a local image with a repository on a registry is
+`username/repository:tag`. The tag is optional, but recommended, since it is
+the mechanism that registries use to give Docker images a version. Give the
+repository and tag meaningful names for the context, such as
+`get-started:part1`. This will put the image in the `get-started` repository and
+tag it as `part1`.
```

Comment:
I think it's correct, given this article: https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375